src/appkit.cs

```diff
@@ -3688,6 +3761,15 @@ public partial interface NSColor : NSCoding, NSCopying, NSSecureCoding, NSPasteb
 		[Static, Export ("quaternaryLabelColor")]
 		NSColor QuaternaryLabelColor { get; }
 		
+		[Mac (10,12)]
+		[Static]
+		[Export ("colorWithDisplayP3Red:green:blue:alpha:")]
+		NSColor ColorWithDisplayP3Red (nfloat red, nfloat green, nfloat blue, nfloat alpha);
+
+		[Mac (10,12)]
+		[Static]
+		[Export ("colorWithColorSpace:hue:saturation:brightness:alpha:")]
+		NSColor ColorWithColorSpace (NSColorSpace space, nfloat hue, nfloat saturation, nfloat brightness, nfloat alpha);
```

Comment:
`WithColorSpace` does not add value (parameter makes it obvious). Maybe just `GetColor` ?